packages/wonder-blocks-testing/src/harness/make-test-harness.js

```diff
@@ -0,0 +1,68 @@
+// @flow
```

Comment:
This is copied from part of the webapp implementation. Basically, the code for `with-test-harness.js` in webapp has been split into:

- make-test-harness.js
- make-hook-harness.js
- test-harness.js
- hook-harness.js

This makes a more versatile and easier to follow API (at least as far as I see it). When we put this back into webapp, we should be able to replace `withTestHarness` implementation with a call to `makeTestHarness`, similiarly, `hookTestHarness` with a call to `makeHookHarness`.